src/runner/runESLint.js

```diff
@@ -122,6 +122,15 @@ const getComputedFixValue = ({ fix, quiet, fixDryRun }) => {
   return undefined;
 };
 
+const removeUndefinedFromObject = object => {
+  return Object.keys(object).forEach(v => {
+    if (object[v] === undefined) {
+      // eslint-disable-next-line no-param-reassign
+      delete object[v];
+    }
+  });
+};
```

Comment:
mutating `extraOptions` doesn't seem like a good idea
```suggestion
function removeUndefinedFromObject(object) {
  return Object.fromEntries(Object.entries(object).filter(([, value]) => typeof value !== 'undefined'));
};
```
and if the current supported node versions don't support Object.fromEntries and/or Object.entries, please use https://npmjs.com/object.fromentries and https://npmjs.com/object.entries.

Reply:
Thanks for the comment!
I have incorporated your suggestion!